rocketpool/node/collectors/odao-collector.go

```diff
@@ -76,16 +76,13 @@ func (collector *OdaoCollector) Collect(channel chan<- prometheus.Metric) {
 	blockNumberFloat := float64(state.ElBlockNumber)
 	pricesBlockFloat := float64(state.NetworkDetails.PricesBlock)
 	effectiveRplStakeBlockFloat := pricesBlockFloat
-	latestReportableBlockFloat := float64(state.NetworkDetails.LatestReportablePricesBlock)
```

Comment:
Taking this out instead of adding a `isHoustonDeployed` check on it will remove it from the Smartnode while Atlas is still live, and we probably want to keep it around during that.